exotic/api/elca.py

```diff
@@ -215,6 +217,15 @@ def getPhase(curTime, pPeriod, tMid):
     return phase - int(np.nanmin(phase))
 
 
+@njit(fastmath=True)
+def mc_a1(m_a2, sig_a2, transit, airmass, data, n=10000):
+    a2 = np.random.normal(m_a2, sig_a2, n)
+    model = transit * np.exp(np.mean(a2) * airmass)
+    detrend = data / model
+    a1 = np.random.normal(np.mean(detrend), np.std(detrend), n)
```

Comment:
It might be better to take the median rather than the mean. The median is a little more robust to outliers

Reply:
Modified them just now to take the median instead of the mean. Thanks!